src/devices/lacrosse_tx22uit.c

```diff
@@ -0,0 +1,135 @@
+/** @file
+    LaCrosse Technology View TX22U-IT temperature, humidity, wind speed/direction
+    and rain sensor
```

Comment:
First lines in doc comments needs to end with a period.

Reply:
will do.